src/Handler/CacheTrait.php

```diff
@@ -121,7 +127,7 @@ protected function cacheResponse(RequestInterface $request, Response $response,
         return $this->cache->set(
             self::getKey($request),
             serialize($cached),
-            $ttl ?: $this->getCachettl($response)
+            $ttl ?: $cacheTtl
```

Comment:
`$ttl` is enough no ? It will contains the right value assigned by `($ttl ?: $cacheTtl)` above